spec/10-overview.md

```diff
@@ -1,16 +1,13 @@
 # Overview
 
-Trace context consists of a set of properties crucial for event correlation in distributed applications. Trace context positions requests in a distributed trace. If you look at a distributed trace as a graph of correlated requests - a single request can be part of multiple graphs. For example, let's say component `A` calls component `B` which in turn calls `C`. One distributed tracing system may see component `A` calling component `B`. Another may see all three components communicating. So the call from `B` to `C` will carry trace context about service `A` as well as information about its direct communication upstream to `B`.
+Trace context is split into two individual propagation fields supporting interoperability and vendor-specifc extensability:  

```

Comment:
this section used to introduce a notion of graphs that will be used later to describe the semantics of `tracestate`. Removing this part, please make sure that proper introduction in the graph notion is in the section explaining `tracestate`